project/api/models/account.py

```diff
@@ -273,16 +273,17 @@ def get_voted_bills(self):
 
         for activity in activities.iterator():
             if activity.is_negative_vote:
-                self._add_linked_civis(supported_bills, activity)
+                if activity.civi.c_type == 'cause':
```

Comment:
Add a comment here describing the intent of the code, since it is a bit ambiguous.

Reply:
Added this as part of docstring